src/screens/SearchResultsScreen.tsx

```diff
@@ -72,7 +71,7 @@ export default function SearchResultsScreen(): JSX.Element {
      * @returns {JSX.Element}
      */
     const searchResultCards = useMemo((): JSX.Element => {
-        const CardComp: React.FC<ShowCardProps> = (props) => {
+        const CardComp: React.FC<ShowCardProps | ShowListCardProps> = (props) => {
```

Comment:
Any reason we don't combine `ShowCardProps` and `ShowListCardProps` into one type because they contain the same properties?

Reply:
I went back and forth on this. What I ended up deciding was that this will make it easier to add different props to each card in the future. With that said, they are identical right now, so it would be reasonable to combine them/just get rid of ShowListCardProps.

Reply:
Makes sense